<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Automatic Synchronization</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>76</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Interval:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="syncIntervalSpinBox">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>[s]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Shortcuts</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="2">
       <widget class="QPushButton" name="resetExitButton">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Synchronize:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="resetOptionsButton">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="resetSyncButton">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="GrabLineEdit" name="deleteContactEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Options:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Edit Contact:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="GrabLineEdit" name="editContactEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Delete Contact:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="GrabLineEdit" name="syncEdit"/>
      </item>
      <item row="11" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>New Contact:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="GrabLineEdit" name="optionsEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="resetNewContactButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="resetDeleteContactButton">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Exit:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="resetEditContactButton">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="GrabLineEdit" name="newContactEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="GrabLineEdit" name="exitEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Export:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="GrabLineEdit" name="exportEdit"/>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="resetExportButton">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Import:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="GrabLineEdit" name="importEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="resetImportButton">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="resetToDefaultButton">
       <property name="text">
        <string>Reset to Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GrabLineEdit</class>
   <extends>QLineEdit</extends>
   <header>GrabLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
